store/carStore.tsx

```diff
@@ -14,21 +15,32 @@ class Cars {
     reserve: null,
     used: false,
     newcar: false,
-    mileage: null,
     file: [],
+    _id: "",
   };
 
   constructor() {
     makeAutoObservable(this);
   }
 
-  addCarInStore = (values: ICar) => {
+  async getCarsInStore() {
+    try {
+      this.isLoading = true;
+      const response = await axios.get(`${BaseUrl.URL}cars/get`);
+      this.cars = [...response.data];
+      this.isLoading = false;
```

Comment:
Move this to `finally` block. In case of any error you will get infinite spinner